source/_components/hive.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title: "Hive"
+description: "Instructions on how to integrate Hive devices with Home Assistant."
+date: 2017-09-24 21:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hive.png
+ha_category: Hub
+ha_release: 0.59
+ha_iot_class: "Cloud Polling"
+---
+
+
+This Hive component is the main component to set up and integrate all supported Hive devices. Once configured with the minimum required details it will detect and add all your Hive devices into Home Assistant, including support for multizone heating.
+
+This component uses the unofficial API used in the official Hive website [https://my.hivehome.com](https://my.hivehome.com), and you will need to use the same Username and Password you use on the Hive website to configure this Hive component in Home Assistant.
+
+To add your Hive devices into your Home Assistant installation, with an update frequency of every five minutes (the default is every two minutes if the scan_interval variable is not specified) add the following to your 'configuration.yaml' file:
```

Comment:
Remove the reference to five minutes. The example doesn't include the scan interval option anymore.

Reply:
done